jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -95,6 +114,58 @@ def generate_paper_letters(self):
             self._app.logger.error(f'Error generating batch letters: {e}')
         self._app.logger.debug('Finish generating batch letters.')
 
+    def upload_to_sftp(self, client, data, filename):
+        """SFTP data to targeted destination."""
+        return client.putfo(
+            fl=BytesIO(data),
+            remotepath=f'{self._app.config.get("BCMAIL_SFTP_STORAGE_DIRECTORY")}/{filename}'
+        )
+
+    def update_notes_and_status(self, furnishings_list, funishing_status, furnishing_notes=None):
```

Comment:
This function will take a furnishings list and change it's status and notes. In this case, I'll using it for MAIL furnishing entries (if the SFTP process fails).